src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -49,7 +49,10 @@ const defaultProps = {
 
 const BankAccountStep = (props) => {
     const shouldReinitializePlaidLink = props.plaidLinkOAuthToken && props.receivedRedirectURI && props.achData.subStep !== CONST.BANK_ACCOUNT.SUBSTEP.MANUAL;
-    const subStep = shouldReinitializePlaidLink ? CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID : props.achData.subStep;
+    let subStep = props.reimbursementAccount && props.reimbursementAccount.achData ? props.reimbursementAccount.achData.subStep : '';
```

Comment:
Here `achData` will be undefined if we're adding a new bank account so we either need to add some default props or this.